docs/digger/installation/install-android-sdk.asciidoc

```diff
@@ -1,4 +1,4 @@
-=== How to install Android SDK with Ansible
+==== How to install Android SDK with Ansible
```

Comment:
I can't see that this file is used in the installation guide but maybe this changes in another pr?

Reply:
@laurafitzgerald it is not included in any guide, was thinking about removing this for idk if it will bring much value to the reader

Reply:
jiraed https://issues.jboss.org/browse/AGDIGGER-264

Reply:
will leave the file in the repo, will not include it in a guide, wdyt @laurafitzgerald 
will rename it to [unused_doc]install-android-sdk.asciidoc

Reply:
resolved